lib/index.js

```diff
@@ -103,7 +91,9 @@
   };
 
   var noTouchHappened = function () {
-    return !touchEvents.touched || new Date().getDate() > touchEvents.lastTouchDate + TOUCH_DELAY;
+    return !touchEvents.touched && (
+      !touchEvents.lastTouchDate || new Date().getTime() > touchEvents.lastTouchDate + TOUCH_DELAY
```

Comment:
If not touch event triggered on this element and no touch event triggered ever, or recently then no touch happened.